tendrl/gluster_integration/central_store/__init__.py

```diff
@@ -44,3 +44,6 @@ def save_nodecontext(self, node_context):
 
     def save_utilization(self, utilization):
         tendrl_ns.etcd_orm.save(utilization)
+
+    def save_job(self, job):
```

Comment:
@shtripat yes job class in common right,  but save method should be in central_store to store that  job object, so i have created job class in common and save method in all component.